pkg/transformer/kubernetes/k8sutils_test.go

```diff
@@ -231,6 +232,97 @@ func TestCreateServiceWithServiceUser(t *testing.T) {
 	}
 }
 
+func TestCreateServiceWithConfigLongSyntax(t *testing.T) {
```

Comment:
@cdrage are the e2e tests still running?

Reply:
they should, looks like it passed on this PR.